src/components/ActionItem.tsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import styled from "styled-components/macro";
-import { ItemText, ItemLeft, Text } from "components";
+import { ItemText, ItemLeft } from "components";
+import { Text } from "components/Text"; // WTF if I import Text from components I get an weird error :/
```

Comment:
what error was that?

Reply:
maybe we can update the comment to point to the fact that we're getting an error because of an outdated version of `styled-component` then? to avoid someone landing on this code in the future and saying `WTF is this WTF`

Reply:
Most probably it should be a circular dependency problem 🤔

Reply:
Strange... I removed the `defaultProps` from `Text` and now it works... : astonished: